storage/innobase/xtrabackup/test/t/bug1394493.sh

```diff
@@ -0,0 +1,103 @@
+############################################################################
+# Bug1394493: need warning when encountering TokuDB tables
+############################################################################
+
+. inc/common.sh
+
+require_server_version_higher_than 5.7.0
+require_xtradb
```

Comment:
We don't really require xtradb, we require tokudb for this test. Lets add function require_tokudb which checks if ha_tokudb.so present.